### DISCUSSION TITLE
Parallel Search in Milvus.

### DISCUSSION THREAD

**Discussion by Yagyansh** [milvus-io/milvus#4723] [Q&A and General discussion]
Does Milvus support parallelism? If yes, how do we measure the parallelism of Milvus and which version has the best parallelism algorithm?

I am currently using Milvus v 0.10.0.

**Comment by shengjun1985** [comment]
What kind of index do you use? How about your row count and 'NQ'?
At present, the graph indexes (HNSW, RNSG) can only be paralleled by 'NQ'; IVF indexes can be paralleled by 'nprobe' or 'NQ'; and FLAT can be paralleled most sufficiently.
The last query performance optimization is in 0.10.2. However, some other bugs and improvement were still fixed later. So, it is recommended to use the latest version 0.10.5
